Nagstamon/Nagstamon/Server/Zabbix.py

```diff
@@ -261,6 +261,11 @@ def _get_status(self):
                     'triggerid': service['triggerid'],
                 }
 
+                if api_version > '3.0':
```

Comment:
So ">=" seems more correct - also better understandable for readers of thid code later if version has to be at least 3.0, regardless of the upcoming minor versions.